views/partials/comments.handlebars

```diff
@@ -1,13 +1,10 @@
 <div class="comments">
      <p class="list-item">Comments</p>
-  {{#each this}}
-  <section class="comment">
-    <div class="meta">
-      Posted by {{user.first_name}} {{user.last_name}} on {{created_at}}
-    <div class="comment">
-      {{comment_text}}
-    </div>
-</div>
-  </section>
+    <ul class= "comments">
+  {{#each posts as |post|}}
```

Comment:
I don't think you need the #each here on the partial for posts, 
Or, the #each needs to be under the #if statement, and re-named to comments